projects/admin/src/app/circulation/item/item.component.html

```diff
@@ -28,7 +28,10 @@
       aria-controls="collapse">
       <i [ngClass]="{ 'fa-caret-down': !isCollapsed, 'fa-caret-right': isCollapsed }" class="fa" aria-hidden="true"></i>
     </button>
-    <a [routerLink]="['/records','items','detail', item.pid]">{{ item.barcode }}</a>
+    <a [routerLink]="['/records','items','detail', item.pid]"
```

Comment:
I would put the id at root (line) level (l.18), because you're targetting only the link. We may need to select the other information of the item, for example to assert that the status is the right one. We could identify the line as `item- +  barcode}`, and the link as a class or a name `barcode`. The select instruction for the barcode link would look like `cy.get(#item-someBarcode .barcode)`